### PR TITLE
Day 3 || Responsive Design

### PR DESCRIPTION
**Pull request by ThorbenBender**

### CODE REVIEW

File: firebase-debug.log [hunk 1]
```diff
@@ -0,0 +1,25 @@
+[debug] [2019-02-06T08:26:06.799Z] ----------------------------------------------------------------------
```
Comment (ivanahuckova) [inline firebase-debug.log:1]: I am not sure if this is necessary in github. If it is, then ignore my comment. If not, then I would add firebase-debug.log to .gitignore.

File: index.html [hunk 2]
```diff
@@ -2,21 +2,18 @@
...
 <head>
-    <title>Art Portfolio</title>
+    <title>Thorben Bender</title>
     <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.1/css/all.css" integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr"
         crossorigin="anonymous">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <link href="css/index.css" rel="stylesheet" />
-    <title>Document</title>
+    <script src="components/navbar.js"></script>
```
Comment (ivanahuckova) [inline index.html:11]: Using components for navbar - good idea!

File: index.html [hunk 3]
```diff
@@ -65,19 +62,22 @@ <h4>Contact</h4>
...
                 <h2>
-                    Hi! I'm Prologue, a free responsive<br>
-                    site template designed by HTML5 UP.
+                    Hi! My name is Thorben Bender<br>
+                    I am a fullstack developer
                 </h2>
                 <p>
-                    Ligula scelerisque justo sem accumsan diam quis<br>
-                    vitae natoque dictum sollicitudin elementum.
+                    On this page, you can see some of the work i done
```
Comment (ivanahuckova) [inline index.html:74]: Just a tiny typo in the text _...of the work I've done_ .

File: index.html [hunk 4]
```diff
@@ -65,19 +62,22 @@ <h4>Contact</h4>
...
+                    and learn more about me.
                 </p>
-                <a href="">Magna Aliquam</a>
+                <a href="#about-me">About Me</a>
```
Comment (ivanahuckova) [inline index.html:77]: I really like that you have decided to use your name and information, so you can use it as a portfolio site.

File: index.html [hunk 5]
```diff
@@ -65,19 +62,22 @@ <h4>Contact</h4>
...
                 <div class="video-container">
                     <video autoplay muted loop id="myVideo">
-                        <source src="video/blurry-night.mp4" type="video/mp4">
+                        <source src="video/Flamez.mp4" type="video/mp4">
```
Comment (ivanahuckova) [inline index.html:80]: Cool video!

File: less/variable.less [hunk 6]
```diff
@@ -26,6 +26,8 @@
 
 
 //media querys
-@tablet: ~"(max-width: 1320px)";
+@tablet: ~"(max-width: 1434px)";
```
Comment (ivanahuckova) [inline less/variable.less:29]: I like that you have done more types of media queries.

**Review by ivanahuckova — APPROVED** [review]

**Review by samarv — APPROVED** [review]

**Pull request merged**